src/pages/PageNotFound.tsx

```diff
@@ -1,13 +1,46 @@
+import { useContext, useEffect } from "react";
+import { useNavigate } from "react-router";
 import images from "src/assets/index";
+import { Button } from "src/components/Button";
+import { ButtonBack } from "src/components/ButtonBack";
+import { ThemeContext } from "src/features/theme/ThemeContext";
 
 export const PageNotFound = () => {
+  const { changeColor } = useContext(ThemeContext);
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    changeColor("#ffffff");
```

Comment:
Add comments as to why the color change is here

Reply:
Because image for error page has white background I change page background to white due to bad look on grey background.